### ISSUE TITLE
create term 'hyperostosis of long bones'

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#11495]
**Preferred term label:**
hyperostosis of long bones

**Synonyms**

**Definition (free text, please give PubMed ID)**
Cortical thickening of the diaphyses of the long bones (from [OMIM 131300](https://omim.org/entry/131300))

**Parent term (use hpo.jax.org/app)**
Hyperostosis  HP:0100774

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Camurati-Engelmann disease [OMIM 131300](https://omim.org/entry/131300)

**Comment**
Could also consider creating a term for each affected long bone, e.g. hyperostosis of the femur, hyperostosis of the tibia, etc.
see Table 3 in [PMID:15326622](https://pubmed.ncbi.nlm.nih.gov/15326622/) for list of most commonly affected bones.

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062

**Comment by pnrobinson** [comment]
@azankl 
Is this different from [Diaphyseal undertubulation  HP:0005019](https://hpo.jax.org/browse/term/HP:0005019) ?

**Comment by azankl** [comment]
@pnrobinson yes, its different. Diaphseal undertubulation is basically a widening of the shaft, but does not say anything about the thickness of the cortex of the bone. Hyperostosis of long bones can be associated with diaphyseal undertubulation, but you can have one without the other. I have also just added 2 more synonyms (endosteal hyperostosis of long bones and cortical thickening of long bones). In addition to [Camurati-Engelmann disease OMIM 131300](https://omim.org/entry/131300), it also occurs in [ endosteal hyperostosis OMIM 144750](https://omim.org/clinicalSynopsis/144750), and probably some others. The whole nomenclature of hyperostosis vs osteosclerosis vs sclerosteosis vs osteopetrosis is a bit confusing and inconsistent...

**Comment by pnrobinson** [comment]
@azankl 
I wonder if then it is a synonym of this: [Thickened cortex of long bones HP:0000935](https://hpo.jax.org/browse/term/HP:0000935) -- maybe then "Long-bone hyperostosis" should be the primary label?
